.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -16,3 +16,20 @@ Bullet points like
 really help with making this more readable.
 
 Fixes/Closes/Related-to #1 (enter issue number, except in rare cases where none exists).
+
+It is also helpful to add an emoji before the commit message to identify the kind of work done on a single commit. See the following suggestions below:
+
+- :bug: - (bug) bug fix
+- :sparkles: - (sparkles) new feature
+- :exclamation: - (exclamation) tests
+- :fire: - (fire) P0
+- :rocket: - (rocket) performance improvements
+- :lipstick: - (lipstick) css/styling
```

Comment:
Is there anything other than lipstick that says "style"?

Reply:
couldn't find any. open to other suggestions but other conventional changelog use either this or the diamond

Reply:
:crayon: (`:crayon:` seems like a good choice)